Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -3044,55 +3044,63 @@ void WebPage::setFooterBannerHeight(int height)
 }
 #endif
 
-void WebPage::takeSnapshot(IntRect snapshotRect, IntSize bitmapSize, SnapshotOptions snapshotOptions, bool sharedBitmapOnly, CompletionHandler<void(std::optional<ImageBufferBackendHandle>&&)>&& completionHandler)
+void WebPage::takeSnapshot(IntRect snapshotRect, IntSize bitmapSize, SnapshotOptions snapshotOptions, CompletionHandler<void(std::optional<ImageBufferBackendHandle>&&, float)>&& completionHandler)
 {
     std::optional<ImageBufferBackendHandle> handle;
     RefPtr coreFrame = m_mainFrame->coreLocalFrame();
     if (!coreFrame) {
-        completionHandler(WTFMove(handle));
+        completionHandler(WTFMove(handle), 1);
```

Comment:
Maybe we should have a `const float SDRHeadroom = 1.0f` somewhere.

Reply:
Switched from `float` to `WebCore::Headroom` so that we can use the `Headroom::None` constant.